app/controllers/users_controller.rb

```diff
@@ -106,6 +106,7 @@ def update_login_details
     @errors = [] 
     success = []
     usr = current_user
+
     # update email
     if params[:email] != "" && params[:email] != current_user.email
```

Comment:
What if  `"  "` was used as input? Might need to use `strip` to remove all leading and trailing white spaces